lang/cs/Org.Apache.REEF.IO.Tests/TestFilePartitionInputDataSet.cs

```diff
@@ -371,5 +394,26 @@ public IEnumerable<Row> Deserialize(string fileFolder)
                 }
             }
         }
+
+        /// <summary>
+        /// read all the files in the set and return byte read one by one
+        /// </summary>
+        /// <param name="filePaths"></param>
+        /// <param name="local"></param>
+        /// <returns></returns>
+        public IEnumerable<Row> Deserialize(ISet<string> filePaths, bool local)
```

Comment:
This is the exact same code as above, but it differs in the inner call. Why not return a `Stream` and pass it to a deserializer?